raiden/utils/secrethash.py

```diff
@@ -0,0 +1,8 @@
+from hashlib import sha256
+
+from raiden.utils.typing import Secret, SecretHash
+
+
+def sha256_secrethash(secret: Secret) -> SecretHash:
+    """Compute the secret hash using sha256."""
+    return SecretHash(sha256(secret).digest())
```

Comment:
Seems simple enough, but a simple test would be nice.

Reply:
Can you add a small test for this?

Reply:
what do you want me to test? the downstream library?

Reply:
done